curriculum/challenges/english/blocks/lecture-understanding-form-validation/6733d3ab69e94b7df7ee91b0.md

```diff
@@ -43,7 +43,7 @@ The second attribute to control how a submission behaves is the `method` attribu
 
 HTTP stands for Hypertext Transfer Protocol and it is used to transfer data over the web.
 
-HTTP methods are used to define the actions that can be performed on resources, such as `GET`, `POST`, `PUT`, `DELETE`, and so on. You will more about these methods in future lectures.
+HTTP methods are used to define the actions that can be performed on resources, such as `GET`, `POST`, `PUT`, `DELETE`, and so on. You will learn more about these methods in future lectures.
```

Comment:
```suggestion
HTTP methods are used to define the actions that can be performed on resources, such as `GET`, `POST`, `PUT`, `DELETE`, and so on. You will learn more about these methods in future lessons.
```

Reply:
lesson is per https://github.com/freeCodeCamp/freeCodeCamp/pull/62414, I do not want an eventual merge conflict to change it back